### DISCUSSION TITLE
Cancelled ❌ Monthly Wallet and Dapp Meeting #28 [October] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#10650] [Wallet and (d)App Council]
# Brief scheduling note

## TLDR

This month's Wallet and Dapp meeting is cancelled ❌ The original date for the meeting was: Wednesday, 11 October (for future reference).

## Reason

There was relatively low attendance in the past few meetings, so I'm reconsidering the need for this meeting going forward.
I received two requests to continue the meeting, which I'm happy to convert into 1:1s:
- https://github.com/celo-org/celo-monorepo/discussions/10553

## What next?

If you were interested in joining this meeting, please consider joining the bi-weekly "[Developer Office Hours](https://celoecosystem.com/office-hours)" instead. They are hosted by the great Developer Relations team at the Celo Foundation.

> Celo Foundation Developer Relations Team hosts Developer Office Hours bi-weekly.
> [Sign up here.](https://share.hsforms.com/15cP85PsIQcq91jdFx2OtcA53n4j)

<img width="740" alt="image" src="https://github.com/celo-org/celo-monorepo/assets/46296830/1510fa7c-9a86-4342-b02f-7155dea71f28">

You can also comment below if you wish this meeting to continue.

**Comment by webx37926** [comment]
I will sign up for the developer relation team
